components/com_content/src/Model/ArticlesModel.php

```diff
@@ -202,6 +202,10 @@ protected function getListQuery()
         $conditionArchived    = ContentComponent::CONDITION_ARCHIVED;
         $conditionUnpublished = ContentComponent::CONDITION_UNPUBLISHED;
 
+        // The query optimizer of both, mariadb and mysql fails to recognize the correct index on large article sets
+        // For these cases, we enforce the usage of the indexes
+        $primaryIndexAppendix = $db->getServerType() === 'mysql' ? " FORCE INDEX(PRIMARY)" : "";
```

Comment:
```suggestion
        $primaryIndexAppendix = $db->getServerType() === 'mysql' ? ' FORCE INDEX(PRIMARY)' : '';
```
PHP CS: Single quotes whenever possible.